src/dissectors/ec_o5logon.c

```diff
@@ -46,6 +65,24 @@ void o5logon_init(void);
 
 /************************************************/
 
+#undef memrchr
+#define memrchr my_memrchr
```

Comment:
That function is not present in eg. OSX so I implemented it. The define saves us on eg. Linux where it **is** present.

I don't understand cmake but I presume this could be dealt with in some more clever way - but this works fine as-is on Linux and OSX.